react/components/SKUSelector/components/SelectVariationMode.tsx

```diff
@@ -41,7 +41,9 @@ function SelectVariationMode(props: VariationSelectModeProps) {
 
   return (
     <div className={styles.skuSelectorSelectContainer}>
+      {/* Warning: don't change this name property because it's used to check on Slider component, to avoid preventDefault of the event trigger by the dropdown */}
```

Comment:
```suggestion
      {/* Warning: don't change the name property. Its value is being checked in the Slider component to avoid a preventDefault call on the event triggered by the dropdown. */}
```